vaccine_feed_ingest/runners/ca/metrolink/normalize.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import logging
+import pathlib
+import sys
+
+from vaccine_feed_ingest_schema import schema  # noqa: E402
+
+logger = logging.getLogger(__name__)
```

Comment:
Since the CLI runs these as scripts in a subprocess, the logger config is not inherited from parent modules. Could you copy a logger config here?

Reply:
I'd be happy to, but honestly I just copied this from an existing normalize.py a few iterations back. Do you have an example handy of someone doing it right?

Reply:
Actually, found one somewhere else, and used this:
```{python}
logging.basicConfig(
    level=logging.INFO,
    format="%(asctime)s %(levelname)s:%(name)s:%(message)s",
    datefmt="%m/%d/%Y %H:%M:%S",
)
logger = logging.getLogger("ca/metrolink/normalize.py")
```